turborepo-tests/helpers/setup_example_test.sh

```diff
@@ -70,7 +70,7 @@ echo "/tmp/" >>".gitignore"
 $package_manager_command
 
 # Simulating the user's first run and dumping logs to a file
-$turbo_command >>./tmp/run-1.txt 2>&1
+$turbo_command 2>&1 | tee ./tmp/run-1.txt
 
 cat ./tmp/run-1.txt
```

Comment:
I think we can remove the `cat` afterwards since we're now sending logs to the file and `stdout`

Reply:
Yes! Still mucking about with other problems being revealed by these `tees`, but thank you for commenting so I don't forget (as I do).